project-base/storefront/package.json

```diff
@@ -20,7 +20,7 @@
         "typecheck": "tsc --pretty --noEmit --project tsconfig.json"
     },
     "dependencies": {
-        "@floating-ui/react-dom-interactions": "0.9.1",
+        "@floating-ui/react": "^0.24.6",
```

Comment:
Let's put specific version 

`"@floating-ui/react": "0.24.6",`

Reply:
I can see this is the current state, probably changed in a later commit.